### DISCUSSION TITLE
What's the meaning of "downloads" and "likes"

### DISCUSSION THREAD

**Discussion by ming535** [huggingface/huggingface.js#414] [Q&A] [answered]
Hi, I was using huggingface hub api, and would like to clarify some details on the API (I haven't found any documentation).
What is "downloads" and "likes" in the following interface?
I am not sure if "downloads" be the total number of downloads or the number of downloads in the last 30 days?
How about "likes"?

```typescript
interface ModelEntry {
    id: string;
    name: string;
    private: boolean;
    gated: false | "auto" | "manual";
    task?: PipelineType;
    likes: number;
    downloads: number;
    updatedAt: Date;
}
```

**Comment by coyotte508** [comment, marked as answer]
downloads are teh downloads for the last 30 days

`likes` are the number of hearts a repo gets
